modules/trivy-operator/README.md

```diff
@@ -0,0 +1,39 @@
+# Purpose
+
```

Comment:
Just a note: I really like trivy and that would be a nice addition to our GitHub Ci/CD around scanning of docker repos and we build.

Great addition!